lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -273,32 +262,106 @@ def podspec(options = nil)
       #             pod 'SSZipArchive'
       #           end
       #
-      # @example  Defining an exclusive target
+      # @example  Defining a test target
       #
       #           target :ZipApp do
       #             pod 'SSZipArchive'
-      #             target :test, :exclusive => true do
+      #             target :ZipAppTests do
       #               pod 'JSONKit'
       #             end
       #           end
       #
       # @return   [void]
       #
-      def target(name, options = {})
-        if options && !options.keys.all? { |key| [:exclusive].include?(key) }
+      def target(name, options = nil)
```

Comment:
> why have options at all? if we raise if they get sent at all?

for people who passed options in the current CocoaPods version. I'll add a TODO to remove the parameter, though, in a few months (it's just to make upgrading friendlier)